app/src/app/app.component.html

```diff
@@ -1,35 +1,29 @@
 <div class="container my-5">
   <div class="row">
     <div class="col">
-      <h1 class="mb-4">Products Table</h1>
-
+      <h1 class="mb-4">Products List</h1>
       <div class="input-group mb-3">
-        <input
-          type="text"
-          class="form-control"
-          placeholder="Search..."
-          (keyup)="search($event)"
-        />
+        <input type="text" class="form-control" placeholder="Search..." (keyup)="search($event)" />
       </div>
-
       <div class="card mb-3" *ngIf="products">
         <div class="card-body">
           <table class="table" style="table-layout: fixed">
             <thead>
               <tr>
-                <th scope="col">Id</th>
-                <th scope="col">Title</th>
-                <th scope="col">Description</th>
-                <th scope="col">Brand</th>
+                <th>Product Id</th>
+                <th>Title</th>
+                <th>Description</th>
+                <th>Brand</th>
+                <th>Display</th>
               </tr>
             </thead>
             <tbody>
               <tr *ngFor="let product of products; index as i">
-                <td>{{ i + 1 + (page - 1) * pageSize }}</td>
```

Comment:
Why has this been changed? this calculation was done to maintain proper ids when various pages were clicked